xds/internal/client/client.go

```diff
@@ -73,7 +70,7 @@ func New(opts Options) (*Client, error) {
 	dopts := append([]grpc.DialOption{opts.Config.Creds}, opts.DialOpts...)
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
-	cc, err := dialFunc(ctx, opts.Config.BalancerName, dopts...)
+	cc, err := grpc.DialContext(ctx, opts.Config.BalancerName, dopts...)
```

Comment:
Did you decide to not change this to `grpc.Dial()`?

Reply:
Sorry, forgot about it.